### ISSUE TITLE
Proper IPv6 support

### ISSUE THREAD

**Issue by Jamesits** [Jamesits/docker-dst-server#7]
Verify needed:

DST Server
- [ ] IPv6 Support
- [ ] Listen on IPv6 address
- [ ] IPv6 only environment

DST Client
- [x] IPv6 Support
- [ ] IPv6 only environment

Klei auth server
- [ ] IPv6 Support
- [ ] IPv6 only environment

Docker
- [ ] Enable IPv6
- [ ] Auto NAT (like IPv4)

**Comment by serpi90** [comment]
You could avoid docker networking and use `network_mode: "host"` (docker-compose)
That way you skip network virtualization and use the underlying host's network.
It's not valid for all use cases, but may solve problems with ipv6

**Comment by Jamesits** [comment]
@serpi90 Yes I know this. Still, there are cases where custom networking is unfeasible (e.g. in a swarm or k8s cluster).

**Comment by Jamesits** [comment]
Waiting for https://github.com/moby/moby/pull/41622
